plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -110,7 +148,7 @@ private void assertPropertiesDefined(Element element) {
 
   private void assertPropertyDefined(String name) {
```

Comment:
To my way of thinking a test that a property is externalized would look on the relevant element and verify that it begins with a %.

This test is finding externalized strings that do begin with a percent are defined in the plugin.properties file.